### PR TITLE
Add optional haptic feedback on LongPressDraggable

### PR DESCRIPTION
**Pull request by eyallen**
Fix for #15665, seemed like a good first one. 

Add a flag to trigger haptic feedback onStart, defaulting to true. Also added a couple unit tests around the issue.

### CODE REVIEW

**Comment by eyallen** [comment]
I signed it.

File: packages/flutter/lib/src/widgets/drag_target.dart [hunk 1]
```diff
@@ -269,12 +270,15 @@ class LongPressDraggable<T> extends Draggable<T> {
     onDragCompleted: onDragCompleted
   );
 
+  /// If haptic feedback should be triggered on drag start.
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/drag_target.dart]: Consider "Whether" rather than "If".

Comment (eyallen) [reply packages/flutter/lib/src/widgets/drag_target.dart]: Works for me.

**Comment by xster** [comment]
Thanks for your contribution

**Pull request merged**